hack/verify-crdgen.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
```

Comment:
Most of the codes are the same as those in generate-crds.sh, right? Maybe combine them to one script with two command options?

Reply:
fix